scripts/quick-helm-install.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
```

Comment:
Do you want to add a command-line parameter that would apply Helm values for reduced resources to this script?
This way developers and/or owners of small clusters can install StackRox from this same script easily.

Reply:
Good idea, I'll do that as well.